lib/Serge/Sync/Plugin/TranslationService/crowdin.pm

```diff
@@ -76,7 +86,17 @@ sub push_ts {
     }
 
     if ($self->{data}->{upload_translations}) {
-        $cli_return = $self->run_crowdin_cli('upload translations', $langs);
+        my @flags = ();
+        if ($self->{data}->{import_duplicates}) {
+            push @flags, q/--import-duplicates/
+        }
+        if ($self->{data}->{import_eq_suggestions}) {
+            push @flags, q/--import-eq-suggestions/
+        }
+        if ($self->{data}->{auto_approve_imported}) {
+            push @flags, q/--auto-approve-imported/
+        }
```

Comment:
I would like to have the command constructed explicitly, without changing run_crowdin_cli. This is how the other plugins are implemented. I do this by my $action = upload translations and then $action .= '--import-duplicates'.

See the mojito implementation.